modules/tinymce/src/core/main/ts/api/textpatterns/TextPatterns.ts

```diff
@@ -1,7 +1,14 @@
-import { RawPattern, RawDynamicPatternsLookup, DynamicPatternContext } from '../../textpatterns/core/PatternTypes';
+import { RawPattern, RawDynamicPatternsLookup, DynamicPatternContext, DynamicPatternsLookup, Pattern, BlockCmdPattern, BlockFormatPattern, InlineCmdPattern, InlinePattern, InlineFormatPattern } from '../../textpatterns/core/PatternTypes';
 
 export {
+  Pattern,
   RawPattern,
+  DynamicPatternsLookup,
   RawDynamicPatternsLookup,
-  DynamicPatternContext
+  DynamicPatternContext,
```

Comment:
The incorrect type on `block` hasn't been fixed in this type by the looks of it. Is that expected?

Reply:
Nope. Completely forgot that part. Got carried away with exposing types. Please hold.

Reply:
That should be added now.